internal/cli/flags.go

```diff
@@ -0,0 +1,24 @@
+package cli
+
+import (
+	"github.com/spf13/pflag"
+	gengoargs "k8s.io/gengo/args"
+)
+
+func flagLogLevel(flags *pflag.FlagSet) {
+	flags.String("log-level", "info", "Configure log level")
+}
```

Comment:
I think this function can be removed and instead included in `flagGeneratorArgs`?

Reply:
`flagGeneratorArgs` is specifically for setting up flags that affect the generator itself.